dipy/workflows/cli.py

```diff
@@ -35,6 +35,7 @@
     "dipy_median_otsu": ("dipy.workflows.segment", "MedianOtsuFlow"),
     "dipy_recobundles": ("dipy.workflows.segment", "RecoBundlesFlow"),
     "dipy_reslice": ("dipy.workflows.align", "ResliceFlow"),
+    "dipy_shm_convert_mrtrix": ("dipy.workflows.io", "ConvertDipyMrtrixFlow"),
```

Comment:
`dipy_shm_convert_mrtrix` : `dipy_sh_convert_mrtrix`

Despite the name of the module being `shm`, `shm` is less identifiable than `sh` to name spherical harmonics, so at least for the workflow, `sh` means more easily recognizable.

Reply:
or `dipy_convert_sh` you might need to add a parameter as a string to select the output basis type

Reply:
Output basis type does not need to be selected because the same conversion function works both ways. It equals its own inverse. I can explain that in the docstring.